lib/vanagon/driver.rb

```diff
@@ -35,6 +35,10 @@ def initialize(platform, project, options = { :configdir => nil, :target => nil,
     def load_engine(engine_type, platform, target)
       if platform.build_hosts
         engine_type = 'hardware'
+      elsif platform.aws_ami
+        engine_type = 'amazon'
+      elsif platform.docker_image
+        engine_type = 'docker'
```

Comment:
Good catch on this; let's try to make sure the commit that fixed this is preserved on its own when the rest of them are squashed.